internal/shakex4/f1600x4_amd64_src.go

```diff
@@ -0,0 +1,148 @@
+// +build ignore
+
+// AVX2 fourway parallelized KeccaK-f[1600].
+
+package main
+
+import (
+	. "github.com/mmcloughlin/avo/build"
+	. "github.com/mmcloughlin/avo/operand"
+)
+
+func main() {
+	ConstraintExpr("amd64")
+
+	TEXT("f1600x4AVX2", 0, "func(state *[100]uint64, rc *[24]uint64)")
```

Comment:
Doesn't look like you are using stack space? In that case you can use the `NOSPLIT` flag.

Reply:
Yes, you’rer right. Thanks.